evalml/automl/automl_search.py

```diff
@@ -411,11 +426,16 @@ def __init__(
         self.pipeline_parameters = (
             pipeline_parameters if pipeline_parameters is not None else {}
         )
+        self.custom_hyperparameters = (
+            custom_hyperparameters if custom_hyperparameters is not None else {}
+        )
         self.search_iteration_plot = None
         self._interrupted = False
         self._frozen_pipeline_parameters = {}
 
         parameters = copy.copy(self.pipeline_parameters)
+        custom_hyperparameters = copy.copy(self.custom_hyperparameters)
```

Comment:
Why do a copy here? We copy `parameters` on the previous line because we then create a modified version where we add the pipeline-level parameters (time series config etc). But we don't do that for the custom hyperparameters, right?

Reply:
Ah thanks, initially I was considering modifying the `custom_hyperparameters` based on other criteria but forgot to change this back.